build_deps/tf_dependency/tf_configure.bzl

```diff
@@ -185,7 +185,7 @@ def _symlink_genrule_for_dir(
 
     genrule = _genrule(
         genrule_name,
-        " && ".join(command),
+        ";\n".join(command),
```

Comment:
Bash on windows and macos have line length limits (4964 and 7937 respectively). TF2.3 added more include files so we finally were pushed over the cliff and our bash shell segfaulted with a very uninformative error.

Quite lucky this was [investigated by the Bazel team a long time ago](https://github.com/bazelbuild/bazel/issues/3636), since it's really not related to Bazel (other than the way genrules expect each copied element to be specified).